crypto3/libs/transpiler/test/circuit1_lpc/gate_argument.sol

```diff
@@ -0,0 +1,74 @@
+
+// SPDX-License-Identifier: Apache-2.0.
+//---------------------------------------------------------------------------//
+// Copyright (c) 2023 Generated by ZKLLVM-transpiler
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
These files were not supposed to be in this PR. I think you unintentionally added some .sol files here.

Reply:
Yeah, sure, I will remove them